README.md

```diff
@@ -5,39 +5,30 @@ Widget for visualizing 3D Radia models in a jupyter notebook
 
 RadiaViewer allows users to render a magnet geometry and solve for its fields in a self-contained widget. The rendering is done by the VTK.js library.
 
-**NOTE**: The current build requires jupyterlab 2.1.0 and jupyterlab-server 1.2.0. 
+**NOTE**: This depends on JupyterLab 3.x
 
 Installation
 ------------
 
 To install use pip:
 
     $ pip install .
-    $ jupyter nbextension install --py --symlink --sys-prefix jupyter_rs_radia
-    $ jupyter nbextension enable --py --sys-prefix jupyter_rs_radia
-
-To install for jupyterlab
-
-    $ jupyter labextension install js
+	$ jupyter labextension install .
 
 For a development installation (requires npm),
 
     $ git clone https://github.com/radiasoft/jupyter-rs-radia.git
     $ cd jupyter-rs-radia
     $ pip install -e .
-    $ jupyter nbextension install --py --symlink --sys-prefix jupyter_rs_radia
-    $ jupyter nbextension enable --py --sys-prefix jupyter_rs_radia
-    $ jupyter labextension install js
+    $ jlpm install:extension
```

Comment:
```jlpm install:extension``` did not work for me:

```
[py3;@v4 jupyter-rs-radia]$ jlpm install:extension
yarn run v1.21.1
$ jupyter labextension develop --overwrite .
setup.py:41: DeprecatedWarning: create_cmdclass is deprecated as of 0.8 and will be removed in 1.0. "
Use `wrap_installers` to handle prebuild steps in cmdclass.
Use `get_data_files` to handle data files.
Use `include_package_data=True` and `MANIFEST.in` for package data.

  data_files_spec=data_files_spec
setup.py:45: DeprecatedWarning: install_npm is deprecated as of 0.8 and will be removed in 1.0. Use `npm_builder` and `wrap_installers`
  install_npm(HERE, build_cmd="build:prod", npm=["jlpm"]),
Installing /home/vagrant/src/mkeilman/jupyter-rs-radia/jupyter_rs_radia/labextension -> jupyter-rs-radia
Building extension in /home/vagrant/src/mkeilman/jupyter-rs-radia/jupyter_rs_radia
An error occurred.
FileNotFoundError: [Errno 2] No such file or directory: '/home/vagrant/src/mkeilman/jupyter-rs-radia/jupyter_rs_radia/package.json'
See the log file for details:  /tmp/jupyterlab-debug-py7p4a4f.log
error Command failed with exit code 1.
```

I'm not sure why it's looking in the jupyter_rs_radia directory.

```jupyter labextension install .``` worked though.